CMakeLists.txt

```diff
@@ -327,6 +327,7 @@ find_package(Boost 1.58.0 REQUIRED COMPONENTS serialization iostreams)
 include_directories(${Boost_INCLUDE_DIRS})
 
 find_package(LLVM REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
```

Comment:
We don't print out the version for our other libraries.

I went back and forth about adding something in the CMakeLists.txt file myself. In the end, I found just looking at `CMakeCache.txt` gave me all the information I needed for debugging though in the case of `find_package(LLVM)` you would have the infer the version from the `LLVM_DIR`. Other config modules end up populating an internal cmake variable `FIND_PACKAGE_MESSAGE_DETAILS_XYZ:INTERNAL` with version information.

For end users, it would probably be more useful to just put a version range check explicitly as they are not likely to want to read all this cmake verbiage.

Reply:
Good point. Will do.